docs/sdks/js-ts/migration.mdx

```diff
@@ -77,11 +77,11 @@ const res = await kwil.getAccount('public_key');
 
 ```
 
-### ActionBuilder and DBBuilder + Friendly Signature Messages
+### Executing Actions + Friendly Signature Messages
 
-The ActionBuilder and DBBuilder classes now require you to chain a `.publicKey()` method to the builder. The `.publicKey()` can receive a hex string or Uint8Array. If using a NEAR public key, you may also pass the Base58 encoded public key with the "ed25519:" prefix.
+The previous version of Kwil used an `ActionBuilder()` class to build and sign transactions. `ActionBuilder()` has been moved internally. Actions can now be executed by using the `kwil.execute()` method, passing an object that matches the `ActionBody` interface and a [KwilSigner](#kwil-signer-class--ed25519-signatures) (see more below).
```

Comment:
Should we still keep action builders available, but just mark them as deprecated?

Reply:
I think we should remove them from the docs, but mark them as deprecated in the package (so they will appear in TypeScript with a strikethrough) and then add a note in the migration section that this replaces the Builders.

My thinking is that we don't want new users to get used to using the builders if we are going to remove them publically next year.

Reply:
Ok.